### DISCUSSION TITLE
use cronie (or something similar) instead of the cron script

### DISCUSSION THREAD

**Discussion by Zoey2936** [nextcloud/all-in-one#2829] [Ideas] [answered]
I think doing that would be better

**Comment by szaimen** [comment]
You mean inside the nextcloud container? Do you have some links available?

**Reply by Zoey2936** [reply]
cronie: https://pkgs.alpinelinux.org/package/v3.18/community/x86_64/cronie
man docs: https://man7.org/linux/man-pages/man8/cron.8.html
running in foreground: `crond -f`

**Reply by szaimen** [reply]
Thanks! :)

Can you please create a PR?

**Reply by Zoey2936** [reply]
yes

**Reply by Zoey2936** [reply, marked as answer]
#2854
